frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -1,39 +1,174 @@
-    <Grid container item direction="column">
-      <Grid item style={{ marginBottom: "22px" }} />
-    </Grid>
-    {/* Column for project quick links */}
-    <Grid container item direction="column" xs={1}>
-      <Card>
-        <Grid
-          container
-          item
-          direction="column"
-          alignItems="center"
-          spacing={1}
-          style={{ padding: "7px 0" }}
-        >
-          <Grid item>
-            <FontAwesomeIcon icon={faGithub} size="3x" />
-          </Grid>
-          <Grid item>
-            <FontAwesomeIcon icon={faSlack} size="3x" />
+const StyledHeadingContainer = styled(Grid)({
+  marginBottom: "24px",
+});
+
+const StyledQLContainer = styled(Grid)({
+  margin: "-8px 16px 16px 0px",
+});
+
+const DisabledItem = ({ name }: { name: string }) => (
+  <Grid item>
+    <Tooltip title={`${capitalize(name)} is disabled`}>
+      <FontAwesomeIcon icon={faLock} size="lg" />
+    </Tooltip>
+  </Grid>
+);
+
+const fetchProject = (project: string): Promise<IClutch.core.project.v1.IProject> =>
+  client
+    .post("/v1/project/getProjects", { projects: [project], excludeDependencies: true })
+    .then(resp => {
+      const { results = {} } = resp.data as IClutch.project.v1.GetProjectsResponse;
+
+      return results[project] ? results[project].project ?? {} : {};
+    });
+
+const Details: React.FC<DetailWorkflowProps> = ({ children, chips }) => {
+  const { projectId } = useParams();
+  const [projectInfo, setProjectInfo] = React.useState<IClutch.core.project.v1.IProject | null>(
+    null
+  );
+  const [metaCards, setMetaCards] = React.useState<CardTyping[]>([]);
+  const [dynamicCards, setDynamicCards] = React.useState<CardTyping[]>([]);
+
+  React.useEffect(() => {
+    if (children) {
+      const tempMetaCards: CardTyping[] = [];
+      const tempDynamicCards: CardTyping[] = [];
+
+      React.Children.forEach(children, child => {
+        if (React.isValidElement(child)) {
+          const { type } = child?.props;
+
+          switch (type) {
+            case CardType.METADATA:
+              tempMetaCards.push(child);
+              break;
+            case CardType.DYNAMIC:
+              tempDynamicCards.push(child);
+              break;
+            default: {
+              if (child.type === DynamicCard) {
+                tempDynamicCards.push(child);
+              } else if (child.type === MetaCard) {
+                tempMetaCards.push(child);
+              }
+              // Do nothing, invalid card
+            }
+          }
+        }
+      });
+
+      setMetaCards(tempMetaCards);
+      setDynamicCards(tempDynamicCards);
+    }
+  }, [children]);
+
+  /**
+   * Takes an array of owner emails and returns the first one capitalized
+   * (ex: ["clutch-team@lyft.com"] -> "Clutch Team")
+   */
+  const getOwner = (owners: string[]): string => {
+    if (owners && owners.length) {
+      const firstOwner = owners[0];
+
+      return firstOwner
+        .replace(/@.*/, "")
```

Comment:
nit: we might want this to be more specific (e.g. if someone were to want to include the @ sign for slack rooms or something). Maybe 
```suggestion
        .replace(/@.*\..*/, "")
```

Reply:
Works for me!